### PR TITLE
Some reverse search fixes

### PR DESCRIPTION
**Pull request by mdashlw**
### Before you begin

* I understand my contributions may be rejected for any reason
* I understand my contributions are for the benefit of Derpibooru and/or the Philomena software
* I understand my contributions are licensed under the GNU AGPLv3

- [x] I understand all of the above

---

### CODE REVIEW

File: lib/philomena/duplicate_reports/search_query.ex [hunk 1]
```diff
@@ -21,7 +21,7 @@ defmodule Philomena.DuplicateReports.SearchQuery do
     search_query
     |> cast(attrs, [:distance, :limit])
     |> validate_number(:distance, greater_than_or_equal_to: 0, less_than_or_equal_to: 1)
-    |> validate_number(:limit, greater_than_or_equal_to: 1, less_than_or_equal_to: 50)
+    |> validate_number(:limit, greater_than_or_equal_to: 1, less_than_or_equal_to: 100)
```
Comment (liamwhite) [inline lib/philomena/duplicate_reports/search_query.ex:24]: 100 isn't allowed as a limit anywhere else, is it?

File: lib/philomena_web/templates/search/reverse/index.html.slime [hunk 2]
```diff
@@ -40,7 +40,6 @@ h1 Reverse Search
 
 = cond do
   - is_nil(@images) ->
-
```
Comment (liamwhite) [inline lib/philomena_web/templates/search/reverse/index.html.slime:43]: Unintended whitespace change?

File: lib/philomena/duplicate_reports/search_query.ex (hunk 1 above)
Comment (mdashlw) [reply lib/philomena/duplicate_reports/search_query.ex:24]: the main issue is that it doesn't sort the results so 50 may not be enough especially with a distance of 1

File: lib/philomena_web/templates/search/reverse/index.html.slime (hunk 2 above)
Comment (mdashlw) [reply lib/philomena_web/templates/search/reverse/index.html.slime:43]: vscode auto formatting

**Pull request merged**